api/scale-of-belief.yml

```diff
@@ -190,6 +257,21 @@ definitions:
         format: uri
       score:
         $ref: '#/definitions/Score'
+  ApiUser:
+      type: object
+      properties:
+        guid:
+          type: string
+          format: guid
+        api_pattern:
+          type: array
+          items:
+            type: string
```

Comment:
If we're using `guid` for `guid`, we could use `regex` here or something similar.